packages/typespec-test/test/contentsafety_modular/generated/typespec-ts/src/models/index.ts

```diff
@@ -2,32 +2,36 @@
 // Licensed under the MIT license.
 
 export {
+  AnalyzeTextOptions,
   TextCategory,
   AnalyzeTextOutputType,
   AnalyzeTextResult,
   TextBlocklistMatchResult,
   TextAnalyzeSeverityResult,
+  AnalyzeImageOptions,
   ImageData,
   ImageCategory,
   AnalyzeImageOutputType,
   AnalyzeImageResult,
   ImageAnalyzeSeverityResult,
   TextBlocklist,
+  AddOrUpdateBlockItemsOptions,
   TextBlockItemInfo,
-  AddBlockItemsResult,
+  AddOrUpdateBlockItemsResult,
   TextBlockItem,
-  CustomPage,
+  RemoveBlockItemsOptions,
+  TextBlocklistResult,
+  TextBlockItemListResult,
 } from "./models.js";
 export {
-  AnalyzeTextOptions,
-  AnalyzeImageOptions,
+  AnalyzeTextOptions as AnalyzeTextOptionsOptions,
+  AnalyzeImageOptions as AnalyzeImageOptionsOptions,
```

Comment:
This is the part where service has model named with Options suffix and they also has operations  named analyzeText or analyzeImage and codegen will give the operation option a name as `PascalCase(${operationName})Options` which is conflict with the model name.  

We could fix it by rename like this, but I prefer to ask service team fix it in the TypeSpec if it's possible.